Sources/Orbit/Components/Button.swift

```diff
@@ -283,6 +283,22 @@ extension Button {
             }
         }
     }
+
+    public struct Content: ExpressibleByStringLiteral {
```

Comment:
Do we want this conformance to be public? Will it be useful outside of previews?

Reply:
Oh wait, there is no such thing as an "internal conformance" is there 🤔

Reply:
Good point - added as internal conformance through extension